README.rst

```diff
@@ -1,24 +1,36 @@
 trac-to-github
 ==============
 
-Hacks used to migrate from Trac sqlite to GitHub.
+Hacks used to migrate from Trac sqlite or Postgresql dump to GitHub.
 
 Works with Python 3.8.
 
 For wiki migration, you will need git available in your dev environment.
 
+This is a 2 stage process:
+
+1. Convert the wiki pages using native Trac wiki content.
```

Comment:
I don't quite get this.
* By "native Trac wiki content", do you mean "native Trac wiki syntax"?
* By "using", do you mean "to"? or maybe "from"?

But I think step 1 does something different (the title below is "Convert to git repo"). Maybe step 1 should be "Convert the Trac wiki as-is into a Git repository with history."